todo_app/templates/index.html

```diff
@@ -25,18 +28,29 @@ <h1 class="display-4">To-Do App</h1>
 <div class="p-2 mx-4 border-black-25 border-bottom"></div>
   <div class="row">
     <div class="col-md-12">
-      <h2>Items</h2>
+      <table>
+        <tr>
+          <th>Title<form action="/sort"><button style="background-color:#535252" type="submit" class="btn btn-primary" formmethod="post" formaction="/sort" value="title" name="sortBy">Sort</button></form></th>
+          <th>Due Date</th>
+          <th>Status<form action="/sort"><button style="background-color:#535252" type="submit" class="btn btn-primary" formmethod="post" formaction="/sort" value="status" name="sortBy">Sort</button></form></th>
+          <th>Action<form action="/sort"><button style="background-color:#535252" type="submit" class="btn btn-primary" formmethod="post" formaction="/sort" value="reset" name="sortBy">Clear Sort</button></form></th>
+        </tr>
       <ul class="list-group mb-4" style="list-style-type:none;">
         <!-- List the todo items here -->
         {% for item in list %}
-        <li>
           <form action="/mark"> 
-            {{item.title}} - <span {% if item.status == 'Completed' %} style="background-color:#00FF00" {% else %} style="background-color:#ff5959" {% endif %}>{{item.status}}</span>
-            <button class="btn btn-success" type="submit" formmethod="post" formaction="/mark" value={{item.id}} name="item_id">Mark Item as 'Complete'</button>
-            <button class="btn btn-danger" type="submit" formmethod="post" formaction="/rem" value={{item.id}} name="item_id">Remove Item</button><br><br>
+            <tr>
```

Comment:
If you are making this a `<tr>`, which stands for "table row", then it belongs in a `<table>`.

"li" stands for "list item" for items in a list, such as "ul", which stands for "unordered list".